### PR TITLE
Added more unit tests + null check to entity report / entity instance report

### PR DESCRIPTION
**Pull request by Mark-de-Haan**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

File: molgenis-dataexplorer/src/main/java/org/molgenis/dataexplorer/controller/DataExplorerController.java [hunk 1]
```diff
@@ -425,21 +425,30 @@ public String viewEntityDetails(@RequestParam(value = "entityTypeId") String ent
...
 			@PathVariable(value = "entityId") String entityId, Model model) throws Exception
 	{
 		EntityType entityType = dataService.getEntityType(entityTypeId);
-		Object id = getTypedValue(entityId, entityType.getIdAttribute());
+		if (entityType == null)
+		{
+			throw new MolgenisDataAccessException(
+					"EntityType with id [" + entityTypeId + "] does not exist. Did you use the correct URL?");
+		}
+		else
```
Comment (ConnorStroomberg) [inline molgenis-dataexplorer/src/main/java/org/molgenis/dataexplorer/controller/DataExplorerController.java]: No else needed , exception stops the method code
